test/e2e/space_autocompletion_test.go

```diff
@@ -47,6 +48,8 @@ func TestAutomaticClusterAssignment(t *testing.T) {
 					testconfig.PerMemberCluster(memberAwait2.ClusterName, -1),
 				),
 		)
+		// some short time to get the cache populated with the change
+		time.Sleep(1 * time.Second)
```

Comment:
why do we need that? what kind of cache needs to be populated?

Reply:
Not sure why, but sometimes the first space was provisioned even after setting -1 to disable cluster capacity on both members, my guess was that ToolChainConfig was not updated with the new values before the space creation was requested. 
I've added more comments on this see [552c9de](https://github.com/codeready-toolchain/toolchain-e2e/pull/656/commits/552c9de9298aecfd678d30c5fe86917526b3e699)